resources/de-amp.js

```diff
@@ -1,57 +1,72 @@
 (function () {
   if (typeof deAmpEnabled !== 'undefined' && deAmpEnabled) {
```

Comment:
this'll still throw in strict mode I believe.  The most robst thing to do here would be something like:

```
let isDeAmpEnabled
try {
    isDeAmpEnabled = !!deAmpEnabled
} catch () {
    isDeAmpEnabled = false
}
```